### DISCUSSION TITLE
Request: Add atlas.hcl skip to postgres extensions

### DISCUSSION THREAD

**Discussion by erickhouse** [ariga/atlas#3052] [General]
We use docker to run `migrate diff`. I have manually installed Postgis in the docker version so the migrations know about the geo types. When I actually go to apply it, they need to be skipped because the cloud version of Postgres we are using has it installed already. 

I want to be able to keep the `create extension` in the schema directory without having generated as part of a` migrate diff`

```
diff {
  skip {
    add_extensions = true
  }
}
```

Perhaps I'm solving the problem incorrectly. Any advice would be helpful. Thanks!

**Comment by a8m** [comment]
The `baseline` schema in the Docker driver was created for cases like this. See: https://atlasgo.io/concepts/dev-database#baseline-schema
